lib/screens/telehealth/chat.dart

```diff
@@ -197,7 +197,7 @@ class _ChatScreenState extends State<ChatScreen> {
                     IconButton(
                       icon: Icon(Icons.photo),
                       onPressed: () async {
-                        File result = await ImagePicker.pickImage(
+                        File result = await ImagePicker.getImage(
```

Comment:
This is not how you fix it
1. Add `final picker = ImagePicker();` at top(~ Line 36)
2. `getImage` does not return a file, it should be used as
```dart
                        final pickedFile = await picker.getImage(
                          source: ImageSource.camera,
                          imageQuality: 80,
                          maxHeight: 400,
                          maxWidth: 400,
                        );

                        if (pickedFile != null) {
                          File result = File(pickedFile.path);
                          uploadImage(result);
                        }
                      },
```